src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -208,6 +208,10 @@ public function load(array $configs, ContainerBuilder $container)
             $this->registerPropertyInfoConfiguration($config['property_info'], $container, $loader);
         }
 
+        if ($this->isConfigEnabled($container, $config['links'])) {
```

Comment:
Link should not be a mandatory component IMO. You mean throwing if enabled is set to `true` but the component not installed?